src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -57,6 +57,9 @@ public sealed class Arguments : DataCommand.ArgumentsBase
 
             [Argument(ArgumentType.Required, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly, HelpText = "Model that needs to be converted to ONNX format.", SortOrder = 10)]
             public ITransformModel Model;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "The targeted ONNX version. It can be either \"Stable\" or \"Experimental\". If \"Experimental\" is used, models produced can contain components not officially defined in ONNX standard.", SortOrder = 11)]
```

Comment:
>models produced can contain components not officially defined in ONNX standard [](start = 164, length = 78)

is used, produced model can contain components that is not officially supported in ONNX standard.